app/assets/stylesheets/wcag.scss

```diff
@@ -7,6 +7,12 @@
   opacity: 1;
 }
 
+:root {
+  --bronze: #d38f42 !important;
+}
+
+$quaternary-high: #c4e9ff;
```

Comment:
Hmm, I hesitate to introduce a color in this specific stylesheet. In core, looks like we only have `$quaternary` and `$quaternary-low`. And the hardcoded color here won't follow the selected color scheme. 

Can you post some screenshots of the issue this is trying to fix?

Reply:
I removed the `$quaternary-high` var, and used a different way to address the issue.